.travis.yml

```diff
@@ -11,3 +11,4 @@ before_install:
   - sudo apt-get update -qq
   - sudo apt-get install libzmq3
   - sudo apt-get install libzmq3-dev
+  - gem update bundler
```

Comment:
Which versions should we test? Currently, followings are stable releases:  
- 2.1.8
- 2.2.4
- 2.3.0